homeassistant/components/openhardwaremonitor/sensor.py

```diff
@@ -30,8 +30,8 @@
 CONF_INTERVAL = "interval"
 
 MIN_TIME_BETWEEN_UPDATES = timedelta(seconds=15)
-SCAN_INTERVAL = timedelta(seconds=30)
-RETRY_INTERVAL = timedelta(seconds=30)
+SCAN_INTERVAL = timedelta(seconds=1)
+RETRY_INTERVAL = timedelta(seconds=1)
```

Comment:
Do you have any documentation or justification for the necessity of this change?

`SCAN_INTERVAL` and `RETRY_INTERVAL` are both now below `MIN_TIME_BETWEEN_UPDATES`

If this is truly necessary, perhaps consider looking at other integrations for inspiration on how to make it configurable.